internal/sql/repository/bulkUpdate/BulkUpdateRepository.go

```diff
@@ -42,34 +42,36 @@ type BulkUpdateRepositoryImpl struct {
 
 func (repositoryImpl BulkUpdateRepositoryImpl) BuildAppNameQuery(appNameIncludes []string, appNameExcludes []string) string {
 	var appNameQuery string
-	var appNameIncludesQuery string
+	appNameIncludesQuery := "app_name LIKE ANY (array["
 	for i, appNameInclude := range appNameIncludes {
 		if i == 0 {
-			appNameIncludesQuery += fmt.Sprintf("app_name LIKE '%s' ", appNameInclude)
+			appNameIncludesQuery += fmt.Sprintf("'%s'", appNameInclude)
```

Comment:
use something like

names := []string{"app", "app2"}
	var fNames []string
	for _, name := range names {
		fNames = append(fNames, fmt.Sprintf("app_name LIKE '%s' ", name))
	}
	fmt.Println(strings.Join(fNames, ","))